pages/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials/index.md

```diff
@@ -100,7 +100,7 @@ account and attach credentials to it.
     <p class="message--note"><strong>NOTE: </strong>Bitbucket Server does not support skipping TLS
     certificate verification for webhooks. If you use a self-signed certificate in your cluster, you
     must [add the certificate to Bitbucket Server](https://confluence.atlassian.com/bitbucketserver/if-you-use-self-signed-certificates-938028692.html).
-    Or if you are using Konvoy, you could [set up a Let's Encrypt certificate](https://docs.d2iq.com/dkp/konvoy/latest/security/letsencrypt/).</p>
+    Or if you are using Konvoy, you could [set up a Let's Encrypt certificate](https://docs.d2iq.com/dkp/kommander/1.4/security/).</p>
```

Comment:
It doesn't seem to me like this new link really helps solve the problem here. That Kommander page doesn't have any information on how to set up a Let's Encrypt certificate.   

Also, this text explicitly mentions 'Konvoy' so it seems like this should still link back to the Konvoy docs.  Also, there *is* still documentation in the Konvoy docs about how to use Let's encrypt to create a certificate, it's here:  https://docs.d2iq.com/dkp/konvoy/1.7/access-authentication/letsencrypt/#create-a-lets-encrypt-certificate

To me, that's a better link, even though that subsection is just a piece of a bigger procedure in the Konvoy docs.

Reply:
Ok, I'm at a loss as to why this link will not work then. Hmmm. Gonna have to dig in further.